photo.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>п
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+  <title>HTML Academy: Седона-страница фото и видео</title>
+</head>
+<body>
+  <header class="main-header">
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">Открыть меню
+      </button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item site-list--active">
+            <a class="site-list__link" href="#">
+              Фото и видео
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              Форма отзыва
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              HTML Academy
+            </a>
+          </li>
+          <li class="site-list__item logo">
+            <a class="logo__link" href="#">
+              <img class="logo__img"
+              src="https://via.placeholder.com/104x83/88cc00/" alt="Логотип «Седона»">
+            </a>
+          </li>
+        </ul>
+      </ul>
+    </div>
+  </nav>
+  <div class="promo promo--base">
+    <img class="promo__img" src="https://via.placeholder.com/229x39/88cc00/"
+    alt="промо-текст Седона">
+  </div>
+</header>
+<main class="page">
+  <div class="page__intro">
+    <h1 class="page__heading">
+      Фото и видео
+    </h1>
+    <p class="page__subheading">Не можете решиться на путешествие из-за курса?
+      Фотографии помогут Вам забыть о  политике и экономике
+    </p>
+  </div>
+  <section class="photo-video">
+    <ul class="photo-list">
+      <li class="photo-list__item">
+        <figure class="photophoto-list__figure">
+          <img  class="photophoto-list__img"
+          src="https://via.placeholder.com/280x280/88cc00/" alt="Фото автора">
+          <figcaption class="photo-list__figcaption">
+            <p>неродные просторы </p>
+            <p> Автор фото: Борис</p>
+          </figcaption>
+        </figure>
+        <div class="like">
+          <button class="like__btn ">
```

Comment:
```suggestion
          <button class="like__btn" type="button">
```
В других подобных кнопках - тоже.